src/main/scala/mesosphere/marathon/api/serialization/ContainerSerializer.scala

```diff
@@ -2,38 +2,61 @@ package mesosphere.marathon.api.serialization
 
 import mesosphere.marathon.Protos
 import mesosphere.marathon.core.externalvolume.ExternalVolumes
-import mesosphere.marathon.state.Container.Docker
 import mesosphere.marathon.state.Container.Docker.PortMapping
 import mesosphere.marathon.state._
-import org.apache.mesos
+import org.apache.mesos.{ Protos => Mesos }
```

Comment:
To carry this out, given the urgency of UC, I will eliminate all changes related to the imports and leave the pure UC-related changes. Then we can create an additional PR for the import cleanups later.